src/sync/installation.ts

```diff
@@ -62,7 +62,11 @@ export const sortedRepos = (repos: Repositories): [string, RepositoryData][] =>
 			new Date(a[1].repository?.updated_at || 0).getTime()
 	);
 
-const getNextTask = async (subscription: Subscription): Promise<Task | undefined> => {
+const getNextTask = async (subscription: Subscription, jiraHost: string): Promise<Task | undefined> => {
+
+	const includeBuildAndDeployments = await booleanFlag(BooleanFlags.BACKFILL_FOR_BUILDS_AND_DEPLOYMENTS, false, jiraHost);
+	const tasks = taskTypes.filter(task => includeBuildAndDeployments ? taskTypes : task !== "build" && task !== "deployment");
```

Comment:
filter need to return true or false.  I think the function you're looking for is `task => includeBuildAndDeployments || task !== "build" && task !== "deployment"`

Reply:
😮‍💨  - fixing